proposals/type-aliases.md

```diff
@@ -2,27 +2,54 @@
 
 * **Type**: Design proposal
 * **Author**: Dmitry Petrov
-* **Contributors**: Andrey Breslav, Stanislav Erokhin
+* **Contributors**: Andrey Breslav, Stanislav Erokhin, Vladimir Reshetnikov
 * **Status**: Under consideration
 * **Prototype**: In progress
 
-## Feedback 
+## Use cases
+
+Type aliases provide alternative names for existing types, for example:
+* Function types
+```
+typealias MyHandler = (Int, String, Any) -> Unit
 
-Discussion of this proposal is held in [this issue](https://github.com/Kotlin/KEEP/issues/4).
```

Comment:
Why is this removed? The link to the relevant issue is needed to direct the feedback there.